brd-android/rockwallet-exchange/src/main/java/com/rockwallet/exchange/ui/features/buy/billingaddress/BillingAddressViewModel.kt

```diff
@@ -177,6 +197,36 @@ class BillingAddressViewModel(
         )
     }
 
+    private fun loadCheckoutApiKey() {
+        callApi(
+            endState = { copy() },
+            startState = { copy(loadingIndicatorVisible = true) },
+            action = { checkoutRiskApi.getApiKeys() },
+            callback = { response ->
+                when (response) {
+                    is ApiResponse.Success -> response.data?.let {
+                        if (it.apiKeys.checkoutApiKey != null && it.apiKeys.isCheckoutProduction != null) {
+                            setEffect {
+                                BillingAddressContract.Effect.SetupRiskJsWebview(
+                                    it.apiKeys.checkoutApiKey!!,
+                                    it.apiKeys.isCheckoutProduction!!
+                                )
+                            }
+                        }
+                    }
```

Comment:
Maybe something like that:

```suggestion
                    is ApiResponse.Success -> response.data?.apiKeys?.let {
                        it.checkoutApiKey?.let { key ->
                            it.isCheckoutProduction?.let { isProduction ->
                                setEffect {
                                    BillingAddressContract.Effect.SetupRiskJsWebview(key, isProduction)
                                }
                            }
                        }
                    }
```